copier.yml

```diff
@@ -14,4 +14,6 @@ min_python:
 max_python:
   type: str
   help: Maximum Python version that is known to work?
+standard_python: '3.10'
+standard_os: 'ubuntu-20.04'
```

Comment:
Can you give these better names, and add a help message? `github_linux_image` and `github_python_version` maybe?

Reply:
I agree that it should have more specific name.
How about `ci_linux_image` and `ci_python_version`...?

Reply:
Is `ci_python_version` ambiguous, as one might think it is the version used for running tests?

Reply:
What exactly do we want to add as a parameter?
A python version for docs build...?

Reply:
This is the Python version used by the runners (for various tasks), *not* the one under which the tests run, isn't it?

Reply:
Yes, but I thought we were using `min_python` for most of runners.
Where exactly should it be used?